src/Microsoft.DotNet.Cli.Utils/Env.cs

```diff
@@ -35,7 +35,10 @@ private static IEnumerable<string> SearchPaths
                 {
                     var searchPaths = new List<string> {AppContext.BaseDirectory};
 
-                    searchPaths.AddRange(Environment.GetEnvironmentVariable("PATH").Split(Path.PathSeparator));
+                    searchPaths.AddRange(Environment
+                        .GetEnvironmentVariable("PATH")
+                        .Split(Path.PathSeparator)
+                        .Select(p => p.Trim('"')));
```

Comment:
Good catch. Thanks! I was able to repro the bug with `pathext` env too. Though it may not be as prevalent as quotes in the `path` env var.

Let me fix this.